src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionRoutesSpec.scala

```diff
@@ -381,7 +381,16 @@ trait GoogleExtensionRoutesSpecHelper extends AnyFlatSpec with Matchers with Sca
       status shouldEqual StatusCodes.Created
       val res = responseAs[UserStatus]
       res.userInfo.userEmail shouldBe user.email
-      res.enabled shouldBe Map("ldap" -> true, "allUsersGroup" -> true, "google" -> true)
+      res.enabled shouldBe TestSupport.enabledMapNoTosAccepted
+    }
+
+    // accept the tos
+    Post("/register/user/v1/termsofservice", TermsOfServiceAcceptance("app.terra.bio/#terms-of-service")) ~> createRoutes.route ~> check {
```

Comment:
OH! Because this is creating a test user for all the `GoogleExtensionRoutesSpec`. The user needs to accept the terms of service. This block is following the paradigm of the block above it, which creates the user, and then makes sure it was successfully created.